backend/assignments/3/src/main/java/com/project/smarthome/dto/request/AddDeviceRequestDto.java

```diff
@@ -0,0 +1,14 @@
+package com.project.smarthome.dto.request;
```

Comment:
wrong placement of files everything is divided into 2 parts response and requests what about internal data transfer?
Either use better approaches like segregating them on better or making structures like

1. exchanges -> (request and response folders)
2. dtos